pkg/csi_driver/node.go

```diff
@@ -49,15 +52,29 @@ type nodeServer struct {
 	mounter     mount.Interface
 	metaService metadata.Service
 	volumeLocks *util.VolumeLocks
+	kubeClient  kubernetes.Interface
 }
 
-func newNodeServer(driver *GCFSDriver, mounter mount.Interface, metaService metadata.Service) csi.NodeServer {
-	return &nodeServer{
+func newNodeServer(driver *GCFSDriver, mounter mount.Interface, metaService metadata.Service, featureLockRelease bool) (csi.NodeServer, error) {
+	ns := &nodeServer{
 		driver:      driver,
 		mounter:     mounter,
 		metaService: metaService,
 		volumeLocks: util.NewVolumeLocks(),
 	}
+	// Create the client which can talk to kube api server.
+	if featureLockRelease {
```

Comment:
should we capture this featureflag in the lock release. that way the feature gated code becomes more readable. e.g where we check for kubeclient != nil, instead we can check feature gate.

Reply:
Done